cms/__init__.py

```diff
@@ -1,6 +1,6 @@
 import django
 
-__version__ = '3.10.0'
+__version__ = 'test'
```

Comment:
This is my fault. I've broken the release script where this is bumped 🤦 

Fix for this here; https://github.com/django-cms/django-cms/pull/7322

Reply:
Should I run the script again, or would it be enough if I just search and replaced the 3.10.0?

Reply:
@creyD Yeah, my fix is merged so I think we'll close this down, delete the release branch from here, you can do the same from your fork. Then run it again and we should see this version as `3.11.0` and the static files bundle in a folder matching the version.